x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -61,6 +61,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       const fieldOptions = await find.allByCssSelector('#thresholdTimeField option');
       await fieldOptions[1].click();
       await nameInput.click();
+      await nameInput.click();
```

Comment:
is this second click needed?

Reply:
If yes, we should put a comment explaining why.